### DISCUSSION TITLE
Error "Cannot read properties of null (reading 'useMemo')"

### DISCUSSION THREAD

**Discussion by sankaSanjeeva** [shadcn-ui/ui#3691] [Q&A]
I am trying to create a library that export a React component. To create that component I used shadcn ui components like Button, Select, Sheet. When I use my library in another project it throws errors like "`Cannot read properties of null (reading 'useMemo')`" if I use 'Select' or 'Sheet' component. If i use only "button" component it works fine.

eg:- this works fine
```
import { Button } from "./components/ui/button";

function App() {
  return (
    <div>
      <h1 className="text-3xl font-bold underline">Hello world!</h1>

      <Button>Click Me</Button>
    </div>
  );
}

export default App;
```

But it throws errors if use Select or Sheet

Appreciate it if someone could tell me whether i am doing something wrong

**Comment by chakrihacker** [comment]
How did you solve this??

**Comment by mwood23** [comment]
Related: https://github.com/denoland/fresh/issues/1530
